packages/core/src/Application/Validator/RenewDomainValidator.ts

```diff
@@ -0,0 +1,62 @@
+import { AbstractValidator, ValidationResult } from 'fluent-ts-validator';
```

Comment:
Do we need extensive validation like this for this use case?

All rules being validated here are already validated by the smart contract, and I can't see how it would improve user experience.

Probably, other than when registering a new domain or TLD, validation should be minimal if existent at all, otherwise we would double the effort to get the same results.